src/components/content/heroes/aurora/weather-easter-egg/use-weather.test.ts

```diff
@@ -0,0 +1,84 @@
+import { renderHook, act } from '@testing-library/react';
```

Comment:
Thank you for educating me. From my "actually I don't know how to test hooks, it's not part of the react docs as far as I know" you found the great testing library and their approach of testing hooks 🤩👍